lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -277,7 +278,11 @@ func (handler *ContainersHandlersImpl) GetContainerListHandler(params containers
 		info := models.ContainerListInfo{}
 		info.ContainerID = &container.ExecConfig.ID
 		info.LayerID = &container.ExecConfig.LayerID
-		info.Created = &container.ExecConfig.Created
+		info.CreateTime = &container.ExecConfig.CreateTime
+		started := container.ExecConfig.Sessions[*info.ContainerID].StartTime
```

Comment:
If we've created a container -- which we have at this point -- the session will always exist, so the StartTime will at a minimum be 0